### PR TITLE
Switch to use renderer library 

### PR DESCRIPTION
**Pull request by jondewijones**
### What

- Switch to use renderer library 

Note: there is a reference to thank you page but it doesn't exist. This is a known bug whereby the thank you template didn't exist in the renderer and so wasn't brought over in this. It's outside the scope of this task and AFAIK no design exists for it

### Who can review

Anyone

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -41,7 +58,9 @@ func Get() (*Config, error) {
 		MailPassword:               "",
 		FeedbackTo:                 "to@gmail.com",
 		FeedbackFrom:               "from@gmail.com",
-		RendererURL:                "http://localhost:20010",
+		SiteDomain:                 "localhost",
```
Comment (nshumoogum) [inline config/config.go:61]: Have we updated the secrets for prod and sandbox?

File: go.mod [hunk 2]
```diff
@@ -3,15 +3,15 @@ module github.com/ONSdigital/dp-frontend-feedback-controller
 go 1.15
```
Comment (nshumoogum) [inline go.mod:3]: Can this be updated to 1.18?

File: handlers/feedback_test.go [hunk 3]
```diff
@@ -265,10 +241,10 @@ func Test_addFeedback(t *testing.T) {
 			addFeedback(w, req, isPositive, mockRenderer, mockSender, from, to, lang)
 
 			Convey("Then the renderer is called to render the feedback page", func() {
-				So(len(mockRenderer.DoCalls()), ShouldEqual, 1)
-				So(mockRenderer.DoCalls()[0].Path, ShouldEqual, "feedback")
-				rendererRequest := string(mockRenderer.DoCalls()[0].B)
-				So(strings.Contains(rendererRequest, `"error_type":"description"`), ShouldBeTrue)
+				So(len(mockRenderer.BuildPageCalls()), ShouldEqual, 1)
+				// So(mockRenderer.DoCalls()[0].Path, ShouldEqual, "feedback")
```
Comment (nshumoogum) [inline handlers/feedback_test.go:245]: Probably worth removing these commented out lines, line 245 to 247)

File: handlers/feedback_test.go [hunk 4]
```diff
@@ -312,10 +289,10 @@ func Test_addFeedback(t *testing.T) {
 			addFeedback(w, req, isPositive, mockRenderer, mockSender, from, to, lang)
 
 			Convey("Then the renderer is called to render the feedback page", func() {
-				So(len(mockRenderer.DoCalls()), ShouldEqual, 1)
-				So(mockRenderer.DoCalls()[0].Path, ShouldEqual, "feedback")
-				rendererRequest := string(mockRenderer.DoCalls()[0].B)
-				So(strings.Contains(rendererRequest, `"error_type":"email"`), ShouldBeTrue)
+				So(len(mockRenderer.BuildPageCalls()), ShouldEqual, 1)
+				//So(mockRenderer.DoCalls()[0].Path, ShouldEqual, "feedback")
```
Comment (nshumoogum) [inline handlers/feedback_test.go:293]: And again

**Review by nshumoogum — CHANGES_REQUESTED** [review]

File: config/config.go (hunk 1 above)
Comment (jondewijones) [reply config/config.go:61]: i haven't as part of this task. assumed it would be another task?

File: go.mod (hunk 2 above)
Comment (jondewijones) [reply go.mod]: 👍

File: handlers/feedback_test.go (hunk 3 above)
Comment (jondewijones) [reply handlers/feedback_test.go]: 👍

File: handlers/feedback_test.go (hunk 4 above)
Comment (jondewijones) [reply handlers/feedback_test.go]: 👍

File: config/config.go (hunk 1 above)
Comment (nshumoogum) [reply config/config.go:61]: Cool that is fine, just wanted to check and I'll create some tickets to do this separately

**Review by nshumoogum — APPROVED** [review]

**Pull request merged**